faq.md

```diff
@@ -141,3 +141,28 @@ to choose a source file whose compile command to use when opening a header.
 The heuristics are currently based on filesystem paths and can sometimes
 choose the wrong source file, though 
 [improvements](https://github.com/clangd/clangd/issues/123) are planned.
+
+## Why does clangd produce false or missing diagnostics?
+
+Most cases of false or missing diagnostics are configuration-related, i.e.
+resulting from clang not using the right compile command for a file.
+However, there are some cases where even files in a correctly configured 
+project may have false or missing diagnostics.
+
+To provide increased responsiveness, clangd skips parsing the bodies of
+functions defined in included headers. This optimization can result in:
+
+- False positive diagnotics, particularly around unused declarations
+  (if the relevant uses are in the code that was skipped). This can be
+  worked around by suppressing affected diagnostic categories in the
+  [config file](https://clangd.llvm.org/config.html#suppress).
+
+- Missing diagnostics, if they occur in code that was skipped. Note
```

Comment:
again there's also some value to keeping entries concise. I don't think the second sentence of this paragraph provides much. Maybe just drop that and keep the link to the issue?